.github/workflows/rust-create-hotfix-pr.yml

```
@@ -0,0 +1,23 @@
+name: Rust Create Hotfix PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Name of the support branch (e.g. `support/v1.2`)'
+        required: true
+
+jobs:
+  create-hotfix-pr:
+    # owner/repository of workflow has to be static, see https://github.community/t/env-variables-in-uses/17466
+    uses: eike-hass/identity.rs/.github/workflows/shared-create-hotfix-pr.yml@dev
```

Should reference the main repository.
```suggestion
    uses: iotaledger/identity.rs/.github/workflows/shared-create-hotfix-pr.yml@dev
```